test/unit/FormulateForm.test.js

```diff
@@ -1,12 +1,19 @@
 import Vue from 'vue'
-import { mount, shallowMount } from '@vue/test-utils'
+import { mount as originalMount } from '@vue/test-utils'
 import flushPromises from 'flush-promises'
 import Formulate from '../../src/Formulate.js'
 import FormSubmission from '../../src/FormSubmission.js'
 import FormulateForm from '@/FormulateForm.vue'
 import FormulateInput from '@/FormulateInput.vue'
 
-Vue.use(Formulate)
+const mount = (app, options) => {
+  const withPlugin = {
+    global: {
+      plugins: [Formulate],
+    },
+  }
+  return originalMount(app, { ...options, ...withPlugin } )
+}
```

Comment:
Plugins are now passed into tests as part of the `global` option in `mount`. This is get around the whole `createLocalVue` issue. A few things have been changed in this area.